pyaedt/modeler/PrimitivesNexxim.py

```diff
@@ -1058,7 +1058,7 @@ def assign_sin_excitation2ports(self, ports, settings):
 
         for prt in ports:
             arg_temp = []
-            arg_temp = ["NAME:EnabledAnalyses", ["NAME:"+str(prt), str(prt)+":=", [] ], ["NAME:VoltageSinusoidal"+str(id), str(prt)+":=", [] ] ]
+            arg_temp = ["NAME:Enabledanalyzes", ["NAME:"+str(prt), str(prt)+":=", [] ], ["NAME:VoltageSinusoidal"+str(id), str(prt)+":=", [] ] ]
```

Comment:
@jleonatti @MaxJPRey  I agree with you. This is a native AEDT command and we cannot modify syntax.